core/src/test/java/nl/nn/adapterframework/testutil/ParameterBuilder.java

```diff
@@ -0,0 +1,36 @@
+package nl.nn.adapterframework.testutil;
+
+import nl.nn.adapterframework.configuration.ConfigurationException;
+import nl.nn.adapterframework.core.ParameterException;
+import nl.nn.adapterframework.core.PipeLineSession;
+import nl.nn.adapterframework.parameters.Parameter;
+import nl.nn.adapterframework.parameters.ParameterList;
+import nl.nn.adapterframework.parameters.ParameterValueList;
+import nl.nn.adapterframework.stream.Message;
+
+public class ParameterBuilder extends Parameter {
```

Comment:
Het kan zo, maar is een beetje confusing, dat de builder de 'buildee' extend.

Reply:
Ja maar vond het beter dan Parameter nog een extra constructor geven. En om nou een parameter veld bij te gaan houden..  was the best of both